index.html

```diff
@@ -4671,140 +4671,168 @@ <h2>Verification Method Rotation</h2>
-binding commitments (e.g., to sign a mortgage).
-<br><br>
-If these conditions are met, revocation is not retroactive; it only nullifies future use of the method.
-<br/><br/>
-However, in order for such semantics to be safe, the second condition &mdash;
-an ability to know what the state of the DID document was at the time the
+        <li>
+The day-to-day operation of any software relying on a compromised
+<a>verification method</a>, such as an individual's operating system, antivirus,
+or endpoint protection software, could be impacted when the <a>verification
+method</a> is publicly revoked.
+      </li>
+
+      <section class="notoc">
+        <h3>Revocation Semantics</h3>
+
+        <p>
+Although verifiers might choose not to accept proofs or signatures from a
+revoked verification method, knowing whether a verification was made with a
+revoked <a>verification method</a> is trickier than it might seem. Some  <a>DID
+methods</a> provide the ability to look back at the state of a <a>DID</a> at a
+point in time, or at a particular version of the <a>DID document</a>. When such
+a feature is combined with a reliable way to determine the time or <a>DID</a>
+version that existed when a cryptographically verifiable statement was made,
+then revocation does not undo that statement. This can be the basis for using
+<a>DIDs</a> to make binding commitments; for example, to sign a mortgage.
+        </p>
+        <p>
+If these conditions are met, revocation is not retroactive; it only nullifies
+future use of the method.
+        </p>
+        <p>
+However, in order for such semantics to be safe, the second condition &mdash; an
+ability to know what the state of the <a>DID document</a> was at the time the
 assertion was made &mdash; is expected to apply. Without that guarantee, someone
-could discover a revoked key and use it to make cryptographically
-verifiable statements with a simulated date in the past.
-<br/><br/>
-Some DID methods only allow the retrieval of the current state of a DID.
-When this is true, or when the state of a DID at the time of
-cryptographically verifiable statement cannot be reliably determined,
-then the only safe interpretation of revocation is to make it
-apply both forward and backward in time. DID ecosystems that
-take this approach essentially provide cryptographically verifiable
-statements as ephemeral tokens that can be invalidated at any time by
-the DID controller.
-      </p>
-    </section>
+could discover a revoked key and use it to make cryptographically verifiable
+statements with a simulated date in the past.
+        </p>
+        <p>
+Some <a>DID methods</a> only allow the retrieval of the current state of a
+<a>DID</a>. When this is true, or when the state of a <a>DID</a> at the time of
+cryptographically verifiable statement cannot be reliably determined, then the
+only safe interpretation of revocation is to make it apply both forward and
```

Comment:
I disagree with the phrase "only safe interpretation" -- because it implies that retroactive cancellation is safe. It isn't. Just like unwise use of revocation when the two constraining safety conditions (knowing when the cryptographic statement was made, and knowing what the state of the DID doc at that instant were), interpreting revocation to apply backward in time ALSO REWRITES HISTORY; the difference is that it rewrites that history to maximize protections for the controller instead of rewriting it to maximize protections for the recipients of cryptographically signed assertions. Either way, there's still a winner and a loser -- a protected party and a party treated cavalierly.

I do agree (strongly) with the third sentence, "DID ecosystems that take this approach..." What I would suggest as a revision would be to say, "When this is true, or when the state of a DID at the time of a cryptographically verifiable statement cannot be reliably determined, then the only safe course is to disallow any consideration DID state with respect to time, except the present moment. DID ecosystems that take this approach..."

What I am trying to do with this revision is to remove the phrase "apply both forward and backward in time" in its link to safety -- because I think describing it that way is problematic.